audio/audio_hw.c

```diff
@@ -1254,10 +1331,31 @@ static int in_set_parameters(struct audio_stream *stream, const char *kvpairs)
     return 0;
 }
 
-static char * in_get_parameters(const struct audio_stream *stream,
+static char* in_get_parameters(const struct audio_stream *stream,
                                 const char *keys)
 {
-    return strdup("");
+    ALOGV("+in_get_parameters(%p) '%s'", stream, keys);
+
+    struct stream_in_common *in = (struct stream_in_common *)stream;
+    struct str_parms *query = str_parms_create_str(keys);
+    struct str_parms *reply = str_parms_create();
+    char value[BUFFER_SIZE];
```

Comment:
Delete. Use str_parms_create_str()

Reply:
But we need to convert const char * to struct str_parms *, which is needed for this  https://github.com/MaksymPrymierov/tinyhal/blob/fix_vts_tests/audio/audio_hw.c#L465 condition.

Reply:
What about this one?